gravitee/values.yaml

```diff
@@ -29,6 +29,42 @@ mongo:
   dbhost: mongo-mongodb-replicaset
   dbname: gravitee
   dbport: 27017
+  auth:
+    enabled: false
+    username:
+    password:
+
+mongodb-replicaset:
+  enabled: true
+  replicas: 3
+  replicaSetName: rs0
+  auth:
+    enabled: false
+    adminUser: username
+    adminPassword: password
+    metricsUser: metrics
```

Comment:
What is this metricsUser ?

Reply:
it's for the mongodb exporter (for prometheus) (if enabled).